src/feature/catalog/product-component.tsx

```diff
@@ -12,7 +12,8 @@ const ProductComponent = ({ poster }: { poster: Poster }) => {
     <div
       key={poster.id}
       className={cn(
-        'group/texthover flex flex-col overflow-visible transition-colors duration-200 hover:rounded-sm',
+        'group/texthover group/cardhover hover:bg-popover/55',
+        'flex flex-col overflow-visible transition-colors duration-200 hover:rounded-sm',
         'shadow-chart-3/15 my-4 rounded-sm shadow-[-8px_8px_16px_-5px]',
```

Comment:
```suggestion
        'group/texthover group/cardhover hover:bg-popover/55',
        'flex flex-col overflow-visible transition-colors duration-200 hover:rounded-sm',
        'shadow-chart-3/15 my-4 rounded-sm shadow-[-8px_8px_16px_-5px]',
        'bg-card',
```
Мб добавим цвет для карточки? А то карточка сливается с задним фоном и границы еле-еле видны
![Анимация2](https://github.com/user-attachments/assets/a695a0d1-0d08-4a81-b4a5-da69653771af)